### PR TITLE
Development v3

### PR DESCRIPTION
**Pull request by smolamarcin**

### CODE REVIEW

File: battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java [hunk 1]
```diff
@@ -96,6 +98,10 @@ private OutputStreamWriter createOutputStream(Socket inputSocket) throws IOExcep
         return new OutputStreamWriter(inputSocket.getOutputStream(), StandardCharsets.UTF_8);
     }
 
+    /**
+     * @return list of all moves send by server
+     * @throws IOException when there was problems during receiving data from server
+     */
     public List<Point2D> receiveMoves() throws IOException {
         String moves = server.receiveMoves();
         if (moves == null || moves.equals("Q")) {
```
Comment (patiem) [inline battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java]: what Q means?

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java [hunk 2]
```diff
@@ -53,7 +53,8 @@ private Parent createContent() {
         enemyBoard = new Board(true, output);
```
Comment (patiem) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java]: true and folse in constructors are like magic number for me, i dont know what they mean

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java [hunk 3]
```diff
@@ -53,7 +53,8 @@ private Parent createContent() {
...
         enemyBoard.initialize(getMove());
         playerBoard = new Board(false, output);
-        shipPlacer = new ShipPlacer(enemyBoard, playerBoard, socketServer, myTurn, socketServer.isFirstPlayer(), waitForSending, output);
+        shipPlacer = new ShipPlacer(enemyBoard, playerBoard, socketServer, myTurn, socketServer.isFirstPlayer(),
```
Comment (patiem) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java]: :D and after refactor you added just another parameter to constructor, sweet

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java [hunk 4]
```diff
@@ -97,10 +98,10 @@ private void informAboutCurrentTurn() {
 
     private void handlePlayersMove(final Cell cell) {
         boolean isMyTurn = cell.shoot();
-        output.send("You shoot at: " + cell.toString());
+        output.send("You shot at: " + cell.toString());
```
Comment (patiem) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java]: String.format()

File: battleship-client/src/main/resources/POLISH.properties [hunk 5]
```diff
@@ -1,7 +1,7 @@
 WELCOME=Witaj w  mojej grze.
 WRONG_INPUT=Nieprawid\u0142owe dane!
-WIN=Wygra\u0142es!
-LOSE=Przegra\u0142es!
+WIN=Wygra\u0142e\u015B!
+LOSE=Przegra\u0142e\u015B!
 INSERT_IP=Podaj IP:
 CONNECT=Po\u0142\u0105cz!
 CLEAR=Wyczy\u015B\u0107 plansz\u0119!
```
Comment (patiem) [inline battleship-client/src/main/resources/POLISH.properties]: :D

File: battleship-server/src/main/java/com/academy/solid/nie/server/Converter.java [hunk 6]
```diff
@@ -5,10 +5,13 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-class Converter {
+final class Converter {
     private static final String SHIPS_SEPARATOR = ";\\|";
```
Comment (patiem) [inline battleship-server/src/main/java/com/academy/solid/nie/server/Converter.java]: <3

File: battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java (hunk 1 above)
Comment (rivalez) [reply battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java]: Q is send when we want to exit game

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java (hunk 2 above)
Comment (rivalez) [reply battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java]: good point

**Pull request merged**